src/app/onprem/license-nag-message-management/nag-message.directive.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * CODENVY CONFIDENTIAL
+ * __________________
+ *
+ *  [2015] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+'use strict';
+
+/**
+ * Defines a directive for displaying licensed nag message.
+ * @author Oleksii Orel
+ */
+export class NagMessage {
+
+  /**
+   * Default constructor that is using resource
+   * @ngInject for Dependency injection
+   */
+  constructor() {
+    this.restrict = 'E';
+    this.replace = false;
+    this.transclude = true;
+    this.bindToController = true;
+    this.controller = 'NagMessageCtrl';
```

Comment:
on a side note we'll have to use Controller in name of controller to avoid errors of jslint which is using
https://github.com/johnpapa/angular-styleguide#controller-names

jslint report  example
 63:5  error  The CheToggleCtrl controller should follow this pattern: /[A-Z].*Controller$/          angular/controller-name

but well, for now we were using Ctrl pattern...